NeuroID/src/release/java/com/neuroid/tracker/storage/NIDDataStoreManager.kt

```diff
@@ -78,14 +78,10 @@ private object NIDDataStoreManagerImp: NIDDataStoreManager {
 
         when (event.type) {
             BLUR -> {
-                CoroutineScope(Dispatchers.IO).launch {
-                    NIDJobServiceManager.sendEventsNow()
-                }
+                NIDJobServiceManager.sendEvents()
             }
             CLOSE_SESSION -> {
-                CoroutineScope(Dispatchers.IO).launch {
-                    NIDJobServiceManager.sendEventsNow(true)
-                }
+                NIDJobServiceManager.sendEvents(true)
```

Comment:
With you previous refactor, can these calls to sendEvents be moved out of the data store manager?